actors/miner/src/lib.rs

```diff
@@ -1630,10 +1630,10 @@ impl Actor {
                 // This is not a CompactCommD, None means that nothing was computed and nothing needs to be checked
                 if let Some(computed_cid) = verify_return.unsealed_cids[i] {
                     // It is possible the computed commd is the zero commd so expand declared_commd
-                    if declared_commd.get_cid(precommit.seal_proof)? != computed_cid {
+                   /* if declared_commd.get_cid(precommit.seal_proof)? != computed_cid {
```

Comment:
Looks like workbench and built-in actors resolve to different computation functions. Workbench mocks this out but built-in Actors is somehow not using the workbench function. As workbench uses this Actors code as WASM code, hard to debug where this is getting resolved to.